src/web/lib/components/ThemeColorsEditor/index.js

```diff
@@ -36,8 +73,8 @@ class ThemeColorsEditor extends React.Component {
             return [
               <li
                 key={`dt-${idx}`}
-                className={classnames(name, "color", { selected: selectedColor === name })}
-                onClick={() => setSelectedColor({ name })}
+                className={classnames(name, "color", { selected: selected && selectedColor === name })}
+                onClick={this.handleClick.bind(this, name)}
```

Comment:
Probably clearer to just use `onClick={() => this.handleClick(name)}` here, since we need to make a closure to pass along `name` every time anyway